### PR TITLE
Add: Announcement Release 32

### PR DESCRIPTION
**Pull request by returntrip**

### CODE REVIEW

File: pages/index.html [hunk 1]
```diff
@@ -3,6 +3,36 @@
...
+
+  <p>Silverblue 32 is now available to <a href="https://silverblue.fedoraproject.org/download">download</a>. Please, install Silverblue 32 and share your thoughts about this release with us!</p>
+
+    <img class="blog-image" alt="Silverblue logo" src="/public/silverblue-logo.svg">
+
+  <p>This release includes some exciting improvements, such as:
+    <ul>
+	  <li>GNOME versions 3.36 - which includes a new Extensions management application (amongst many other improvements)
+	  <li>EarlyOOM - Early Out of Memory Manager is now enabled by default
+		<li>FSTrim - file system trimming service (used to discard all unused blocks in the filesystem) is now enabled by default
```
Comment (dustymabe) [inline pages/index.html:18]: extra indentation ?

**Comment by dustymabe** [comment]
one small comment

File: pages/index.html (hunk 1 above)
Comment (returntrip) [reply pages/index.html]: tab is my enemy :)

**Comment by tpopela** [comment]
Does the fstrim really work? Re. https://github.com/fedora-silverblue/issue-tracker/issues/31 and https://bugzilla.redhat.com/show_bug.cgi?id=1801539. I had to manually edit the kargs to get it working.

**Comment by returntrip** [comment]
> Does the fstrim really work? Re. [fedora-silverblue/issue-tracker#31](https://github.com/fedora-silverblue/issue-tracker/issues/31) and https://bugzilla.redhat.com/show_bug.cgi?id=1801539. I had to manually edit the kargs to get it working.

I had to do the same but forgotten about it. In this case I will remove FSTrim. Anything else we could add?

**Comment by tpopela** [comment]
We now have the default set of applications that are being preinstalled as Flatpaks by Anaconda. The list is still not the same as on Workstation, but we will get there.

**Comment by returntrip** [comment]
> We now have the default set of applications that are being preinstalled as Flatpaks by Anaconda. The list is still not the same as on Workstation, but we will get there.

Just to clarify. Would you like me to add this sentence to the bullet points?

**Comment by tpopela** [comment]
> > We now have the default set of applications that are being preinstalled as Flatpaks by Anaconda. The list is still not the same as on Workstation, but we will get there.
> 
> Just to clarify. Would you like me to add this sentence to the bullet points?

No, definitely not! I quickly replied when I had a dinner. :D Let's use:

"Several Flatpak applications are now being preinstalled by Anaconda to improve the out of the box experience."

We should mention Anaconda there, because the Flatpaks are only preinstalled if a user is doing a clean installation.

**Comment by returntrip** [comment]
Great 🙂. Is there a list of apps being pre installed by Anaconda? I would like to know cause I have not installed Silverblue 32 from scratch. Maybe we could also link the app list to the Announcement?

**Comment by tpopela** [comment]
Currently it's:

Disks
Calculator
Calendar
Characters
Clocks
Contacts
Image Viewer
Document Viewer
Font Viewer
Text Editor
Logs
Maps
Screenshot
Weather
Archive Manager

**Comment by returntrip** [comment]
Ok, pushed an amended version. I have linked the Flatpaks listed above to the announcement, if you think we should remove it let me know, thanks.

**Comment by dustymabe** [comment]
Any reason not to merge?

**Comment by tpopela** [comment]
Thank you @returntrip !

**Pull request merged**